tests/test_sql_query.py

```diff
@@ -335,6 +334,8 @@ async def test_sql_query_24(self):
         self.assertEqual(res, [['Forrest Gump']])
 
     async def test_sql_query_25(self):
+        import asyncpg
```

Comment:
Just guard the import at the top level with `try: except ImportError`.  These tests are not reachable if asyncpg is not available because of the guard in setup.

Reply:
OK will do! Looks like flake8 is not complaining that `asyncpg` may be an undefined name.